.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,30 @@
+Hi there!
+
+Thanks for submitting an issue to Protractor.
+
+To help us help you better, please do the following before submitting an issue:
+
+1. Review the questions section of [CONTRIBUTING.md](https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#contributing)
+2. Ensure that your issue is a bug or a feature request, and not a usage or debugging question
```

Comment:
There are other categories than bug reports and feature request (e.g. documentation).  Maybe just say "Make sure you are not asking a debugging question.  Ask those on StackOverflow, our Google Group discussion list, or Gitter."